ow_power_system/src/power_system_node.cpp

```diff
@@ -152,18 +151,16 @@ bool PowerSystemNode::initTopics()
   m_state_of_charge_pub = m_nh.advertise<Float64>("power_system_node/state_of_charge", 1);
   m_remaining_useful_life_pub = m_nh.advertise<Int16>("power_system_node/remaining_useful_life", 1);
   m_battery_temperature_pub = m_nh.advertise<Float64>("power_system_node/battery_temperature", 1);
-
   // Finally subscribe to the joint_states to estimate the mechanical power
   m_joint_states_sub = m_nh.subscribe("/joint_states", 1, &PowerSystemNode::jointStatesCb, this);
-
   return true;
 }
 
 void PowerSystemNode::jointStatesCb(const sensor_msgs::JointStateConstPtr& msg)
 {
   auto power_watts = 0.0;  // This includes the arm + antenna
   for (auto i = 0; i < ow_lander::NUM_JOINTS; ++i)
-    power_watts = msg->velocity[i] * msg->effort[i];
+    power_watts += fabs(msg->velocity[i]) * fabs(msg->effort[i]);
```

Comment:
Why not just make a single call to fabs here around the entire product?

Reply:
addressed